features/support/fetch_registration_data_helpers.rb

```diff
@@ -0,0 +1,35 @@
+# Used for caching
```

Comment:
I've never used `@@` before (and possibly any other tester looking at the test suite). My understanding is that this is a way of preserving a variable between calls to each method. But I don't understand why `@` wouldn't work here.

Is it possible to add a bit more detail here about how the caching works with this syntax?

Reply:
It can work with @ only as well, will refactor (also because rubocop doesn't like it either). BTW, @@ are class-level variables :)

Reply:
```suggestion
```